### DISCUSSION TITLE
More exposure and share this plugin with more users

### DISCUSSION THREAD

**Discussion by kohane27** [TheSafdarAwan/find-extender.nvim#7] [General]
Ideas to get more exposure and share this plugin with more users:

- create Reddit update post in [r/neovim](https://www.reddit.com/r/neovim/)
- submit a PR to get `find-extender` featured in [This Week in Neovim](https://this-week-in-neovim.org)
- submit a PR to [Awesome Neovim](https://github.com/rockerBOO/awesome-neovim)
- improving README.md (I can help write a 2-minute pitch and some general improvements if you'd like)

If you think the alpha is ready, may be we can proceed with the above suggestions? What do you think?

**Comment by theSafdarAwan** [comment]
> Ideas to get more exposure and share this plugin with more users:
> 
>     * create Reddit update post in [r/neovim](https://www.reddit.com/r/neovim/)
> 
>     * submit a PR to get `find-extender` featured in [This Week in Neovim](https://this-week-in-neovim.org)
> 
>     * submit a PR to [Awesome Neovim](https://github.com/rockerBOO/awesome-neovim)
> 
>     * improving README.md (I can help write a 2-minute pitch and some general improvements if you'd like)
> 
> 
> If you think the alpha is ready, may be we can proceed with the above suggestions? What do you think?

yes i would really appreciate if you create a demo i have broken computer screen. Which doesn't allow me to record things properly and i have to set my windows on half of the screen because other half is not visible.
And yes i think alpha is ready the only thing i want is to add the `count` feature which will allow users to move using counts if there are 4 matches and user is on the first user can use the `3l` to jump to the 3rd match.
Also the reason why i am not sharing, on reddit or in neovim weekly is because there are thing that we won't be able to change if the user base increases lets say 10 users start using this plugin then. Changing things would be not a good idea.
I think after adding the line movements using `jk` keys this would attract more people because i don't think any other plugin does this. And I would really appreciate if you created a demo because currently i am not able to do so because of my Monitor. Thanks 💕.

**Reply by kohane27** [reply]
I'm sorry to hear about your computer screen situation (also let me know when the github sponsorship is ready) Please let me know when `alpha` is ready and I'll make some demo for the README.

**Comment by theSafdarAwan** [comment]
@kohane27 hey, i have been trying to solve a bug for about 4 hours now and its driving me crazy. Can you go the this [line](https://github.com/TheSafdarAwan/find-extender.nvim/blob/f8094a6ff2aca14cd645e7145e626d340e80d2a5/lua/find-extender/finder.lua#L97) in you neovim after updating the find-extender to latest commit. And try finding `st` using lh movement and see if the cursor does not gets rendered.
I have no idea why cursor doesn't gets updated only on this line and now i don't have enough left in me to debug this today thanks!.

**Reply by theSafdarAwan** [reply]
> @kohane27 hey, i have been trying to solve a bug for about 4 hours now and its driving me crazy. Can you go the this [line](https://github.com/TheSafdarAwan/find-extender.nvim/blob/f8094a6ff2aca14cd645e7145e626d340e80d2a5/lua/find-extender/finder.lua#L97) in you neovim after updating the find-extender to latest commit. And try finding `st` using lh movement and see if the cursor does not gets rendered. I have no idea why cursor doesn't gets updated only on this line and now i don't have enough left in me to debug this today thanks!.

ok, so i found the bug.
